ui/pages/routes/routes.component.js

```diff
@@ -760,6 +760,23 @@ export default class Routes extends Component {
             onAutoHideToast={onAutoHideToast}
           />
         ) : null}
+
+        {process.env.CHAIN_PERMISSIONS ?
+          <Toast
+            key="switched-network-toast"
+            startAdornment={
+              <AvatarNetwork
+                size={AvatarAccountSize.Md}
+                borderColor={BorderColor.transparent}
+                src=""
+                name="hhhh"
```

Comment:
Is this `name` valid?

Reply:
Oh! This PR needs to be closed. Completed via Editing Flow PR